src/ImageSharp/Common/Helpers/Vector256Utilities.cs

```diff
@@ -21,24 +20,6 @@ namespace SixLabors.ImageSharp.Common.Helpers;
 internal static class Vector256_
```

Comment:
This could profit from the new c# extension constructs.

Reply:
Yeah, I'm interested in them. I'm hoping many of these methods will eventually make it into the runtime so I don't actually need them though.